nix/mobile/android/release.nix

```diff
@@ -171,12 +173,17 @@ in stdenv.mkDerivation rec {
     ${adhocEnvVars} ${gradleCommand}
     popd > /dev/null
   '';
+
   doCheck = true;
-  checkPhase = ''
+
+  checkPhase = if buildType == "debug" then ''
+    echo "skipping checkPhase for debug builds"
+  '' else ''
     ls ${apksPath}/*.apk \
-      | xargs -n1 ${pkgs.unzip}/bin/unzip -qql \
-      | grep 'index.android.bundle'
+    | xargs -n1 ${pkgs.unzip}/bin/unzip -qql \
+    | grep 'index.android.bundle'
```

Comment:
You have the unused `doCheck` there, use it:
```nix
  doCheck = buildType != "debug";
  checkPhase =  ''
    ls ${apksPath}/*.apk \
      | xargs -n1 ${pkgs.unzip}/bin/unzip -qql \
      | grep 'index.android.bundle'
  ''
```
And please don't remove the indent from split lines lines.

Reply:
ah did not know thats what `doCheck ` does.
Thanks for explaining :)

Reply:
addressed with : https://github.com/status-im/status-mobile/pull/18784/commits/8a330cc8708d68b58e1ab943eb1f06dfe0c4e46d